<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 100, 139);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="li_name">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="li_surname">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="li_email">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>260</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="B_save">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>490</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt &quot;Montserrat SemiBold&quot;;
border-radius: 20 </string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="B_exit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>500</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
font: 63 14pt &quot;Montserrat SemiBold&quot;;
border-radius: 20;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="la_name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 100, 139);
font: 63 14pt &quot;Montserrat SemiBold&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="la_surname">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 100, 139);
background-color: rgb(255, 255, 255);
font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string> Surname</string>
    </property>
   </widget>
   <widget class="QLabel" name="la_mail">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 100, 139);
background-color: rgb(255, 255, 255);
font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string> E-Mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="la_accnumber">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 100, 139);
background-color: rgb(255, 255, 255);
font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string> Account Number</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_allcustomers">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt &quot;Montserrat SemiBold&quot;;
border-radius: 20 </string>
    </property>
    <property name="text">
     <string>All Customers</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>611</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 63 18pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string> ADDING CUSTOMER SCREEN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="la_curbalance">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 100, 139);
background-color: rgb(255, 255, 255);
font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string> Current Balance</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="li_balance">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
    <property name="inputMask">
     <string>999999999999</string>
    </property>
   </widget>
   <widget class="QLabel" name="la_password">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 100, 139);
background-color: rgb(255, 255, 255);
font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="li_password">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>410</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
    <property name="inputMask">
     <string>9999999</string>
    </property>
    <property name="maxLength">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLabel" name="la_error">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>330</y>
      <width>221</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="la_id">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
